assets/src/application/services/utilities/predicates/removeNullAndUndefined/index.ts

```diff
@@ -0,0 +1,7 @@
+import { filter, compose, not, isNil } from 'ramda';
+
+type Obj = { [key: string]: any };
+
+const removeNullAndUndefined = <T>(filterable: T[] | Obj): T[] | Obj => filter(compose(not, isNil), filterable);
```

Comment:
Types here are not good, how can two types be same when some of its props have been removed? It's only possible if return type is `Partial<T>` .